cmd/microservice-sui-user-actions/state.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/block-vision/sui-go-sdk/models"
+	"github.com/block-vision/sui-go-sdk/sui"
+
+	"github.com/fluidity-money/fluidity-app/lib/log"
+	"github.com/fluidity-money/fluidity-app/lib/state"
+	"github.com/fluidity-money/fluidity-app/lib/types/misc"
+)
+
+// getTransactionBlocks to get all blocks from a list of digests, batching requests if there are more than the RPC's limit
+func getTransactionBlocks(client sui.ISuiAPI, digests []string) (models.SuiMultiGetTransactionBlocksResponse, error) {
+	const BatchLimit = 50
```

Comment:
Bring this up into a constant in the main code, and explain the interrelationship